test/ptp/ptp.go

```diff
@@ -30,6 +30,14 @@ import (
 	"github.com/openshift/ptp-operator/test/utils/pods"
 )
 
+var isSingleNode bool
+
+func init() {
+	var err error
+	isSingleNode, err = nodes.IsSingleNode()
```

Comment:
Move the call inside the tests (beforeAll for example)

Reply:
Done